board.c

```diff
@@ -1317,6 +1319,8 @@ board_play_in_eye(struct board *board, struct move *m, int f)
 	board_at(board, coord) = color;
 	group_t group = new_group(board, coord);
 
+	board->last_move4 = board->last_move3;
```

Comment:
dcnn need to use last 4 moves, but I find previous last_move3 and last_move4 is not set. I do not know if this modify is ok?